lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -215,7 +215,7 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
     $materials['title'] = t('Stuff You Need');
     $materials['content'] = $campaign->items_needed;
     // if organ donation campaign, add in the share link here
-    if ($campaign->variables['register_organ_donors']) {
+    if (array_key_exists('register_organ_donors', $campaign->variables) && $campaign->variables['register_organ_donors']) {
```

Comment:
in this case isset by itself should also work fine, i could try it out if you want